Sources/Sentry/Public/SentryMeasurementUnit.h

```diff
@@ -0,0 +1,104 @@
+#import "SentryDefines.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(MeasurementUnit)
```

Comment:
l: Isn't this name too generic?
Thinking about avoid collision in the future.

Reply:
Hmm, yes we could keep the Sentry prefix for every one of these.

Reply:
But do you really think that `MeasurementUnit` is used that often? Otherwise, the names get pretty long. `SentryMeasurementUnitDuration` is very long.

Reply:
TBH Im not sure. Im just playing safe here. I agree that the long name is bad.